docs/reference/monitoring/configuring-monitoring.asciidoc

```diff
@@ -13,10 +13,11 @@ indices. You can also adjust how monitoring data is displayed.
 
 . To collect monitoring data about your {es} cluster:
 
-.. Verify that the `xpack.monitoring.enabled` and 
-`xpack.monitoring.collection.enabled` settings are `true` on each node in the 
-cluster. By default, data collection is disabled. For more information, see 
-<<monitoring-settings>>.
+.. Verify that the `xpack.monitoring.enabled`, 
+`xpack.monitoring.collection.enabled`, and 
+`xpack.monitoring.elasticsearch.collection.enabled` settings are `true` on each
+node in the  cluster. By default, data collection is disabled. For more
```

Comment:
This second sentence is very confusing with this new setting in the mix.

> By default `xpack.monitoring.collection.enabled` is disabled (`false`), and that overrides `xpack.monitoring.elasticsearch.collection.enabled`, which defaults to being enabled (`true`). Both settings can be set dynamically at runtime.

(Or similar)